Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox
+
+COPY migrations /migrations
+
+VOLUME /migrations
```

Comment:
I'm assuming this will see iteration as you're working on the Jenkinsfile, since this won't do much for now, heh

Reply:
I honestly don't expect much iteration on the Dockerfile as I'm working in the Jenkinsfile. My current plan is only to have the migrations stored in the Docker image. I briefly considered using `migrate/migrate` as the base image, but I'd like to avoid coupling our migration script snapshots to a specific version of `migrate`.